packages/vite/README.md

```diff
@@ -2,5 +2,265 @@
 
 The Vite plugin for UnoCSS. Ships with the `unocss` package.
 
-> This plugin does not come with any default presets. You are are building a meta framework on top of UnoCSS, see [this file](https://github.com/antfu/unocss/blob/main/packages/unocss/src/vite.ts) for an example to bind the default presets.
+> This plugin does not come with any default presets. You are building a meta framework on top of UnoCSS, see [this file](https://github.com/antfu/unocss/blob/main/packages/unocss/src/vite.ts) for an example to bind the default presets.
 
+## Installation
+
+```bash
+npm i -D unocss
+```
+
+```ts
+// vite.config.ts
+import Unocss from 'unocss/vite'
+
+export default {
+  plugins: [
+    Unocss({ /* options */ })
+  ]
+}
+```
+
+Add `uno.css` to your main entry:
+
+```ts
+// main.ts
+import 'uno.css'
+```
+
+## Modes
+
+The Vite plugin comes with a set of modes that enable different behaviors.
+
+### global (default)
+
+This is the default mode for the plugin: in this mode you need to add the import of `uno.css` on your entry point.
+
+This mode enables a set of Vite plugins for `build` and for `dev` with `HMR` support.
+
+The generated `css` will be a global stylesheet injected on the `index.html`.
+
+### vue-scoped
+
+`WIP`: coming soon.
```

Comment:
You can copy the desc in jsdoc

Reply:
I add WIP to all modes, I need to collect more info and also add some fixture for some mode.

I also need to review `react` using `@vitejs/plugin-react-refresh` instead `@vitejs/plugin-react`  plugin and `preact` fixtures (when added).